pkg/openstackbaremetalset/baremetalhost.go

```diff
@@ -217,7 +217,7 @@ func BaremetalHostProvision(
 		if foundBaremetalHost.Status.Provisioning.State != metal3v1.StateProvisioned {
 			foundBaremetalHost.Spec.Image = &metal3v1.Image{
 				URL:          localImageURL,
-				Checksum:     fmt.Sprintf("%s.sha256", localImageURL),
```

Comment:
Should the default for OSImageChecksumSuffix be `sha256`?

Or:
 * do we want to override the setting in a separate patch?
 * do we want to change the name of file in the webserver?

```
[ 1038.534029] ironic-python-agent[551]: 2024-05-07 08:04:40.267 551 ERROR root ironic_python_agent.errors.ImageDownloadError: Error downloading image: Download of image http://192.168.126.11:6190/edpm-hardened-uefi.qcow2.sha256sum failed: Received status code 404 from http://192.168.126.11:6190/edpm-hardened-uefi.qcow2.sha256sum, expected 200. Response body: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
```